media-libs/libdmtx/libdmtx-0.7.4.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Library to read and write Data Matrix barcodes"
+DESCRIPTION="libdmtx is a software library that enables programs to read and \
+write Data Matrix (or DataMatrix) 2D barcodes. It runs on several platforms, \
+and can be accessed by multiple languages."
+HOMEPAGE="http://libmtx.sourceforge.net/"
+COPYRIGHT="2005-2011 Mike Laughton"
+LICENSE="libdmtx v1.0"
+REVISION="1"
+SOURCE_URI="https://download.sourceforge.net/libdmtx/libdmtx-$portVersion.tar.gz"
+CHECKSUM_SHA256="686bcfb44e50b96345aa53f41e285f2118a2cf87a333b69b74165a2e7ddca492"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
```

Comment:
Please add
```
SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```
below ARCHITECTURES, otherwise it won't be possible to build recipe for secondary architectures.